test/fuzz/transformation_add_parameter_test.cpp

```diff
@@ -240,7 +239,576 @@ TEST(TransformationAddParameterTest, BasicTest) {
                OpReturn
                OpFunctionEnd
   )";
+  ASSERT_TRUE(IsEqual(env, expected_shader, context.get()));
+}
+
+TEST(TransformationAddParameterTest, NonPointerNotApplicableTest) {
+  // This types handles case of adding a new parameter of a non-pointer type
+  // where the transformation is not applicable.
```

Comment:
Add a test case for the fact that `call_parameter_id` should have an entry for _at least_ every caller (e.i. add some non-caller entries to the map).